stream/pbf.js

```diff
@@ -0,0 +1,68 @@
+
+/**
+  The pbf parser is responsible for configuring and executing a pbf parser and
+  returning a valid readable stream.
+**/
+
+var fs = require('fs'),
+    pbf2json = require('pbf2json'),
+    settings = require('pelias-config').generate(),
+    features = require('../config/features');
+
+// Create a new pbf parser stream
+function createPbfStream( opts ){
```

Comment:
Sure, we can discuss later. But I made the comment in this place specifically because opts is obscuring the interface and even if we don't decide to add doc blocks across the board, a comment here about what opts is expected to have is something I think is sorely missing.